common/dmfsnmp.c

```diff
@@ -515,6 +515,20 @@ mibdmf_get_aux_list_ptr(mibdmf_parser_t *dmp)
 	return &(dmp->list[dmp->sublist_elements - 1]);
 }
 
+alist_t **
+mibdmf_get_initial_list_ptr(mibdmf_parser_t *dmp)
+{
+        if (dmp==NULL) return NULL;
+        return dmp->list;
+}
+
+int
+mibdmf_get_list_size(mibdmf_parser_t *dmp)
```

Comment:
By design could me NULL, is going a be checked and initialized in another part of the program, but thanks